### DISCUSSION TITLE
Developer Site Login Details

### DISCUSSION THREAD

**Discussion by jmerriweather** [amberelectric/public-api#15] [General]
Hi,

I've used the API previously and received an email regarding the new version, however I don't believe I've ever had a developer login. Do we use our normal Amber credentials? or should we have been setup with developer login details?

I've attempted the reset password but nothing comes through.

Kind regards,
Jonathan

**Comment by madpilot** [comment]
Yep, just use your regular Amber login details

**Comment by jmerriweather** [comment]
I found the problem, the email field to login to Amber is case sensitive. Can we report that as a bug somewhere?

**Reply by madpilot** [reply]
I'll put it in our backlog tomorrow
